core/schema/modulesource_test.go

```diff
@@ -91,6 +242,37 @@ func TestParsePublicRefString(t *testing.T) {
 				kind:           core.ModuleSourceKindGit,
 				repoRoot:       &vcs.RepoRoot{Root: "bitbucket.org/test-travail/test.git", Repo: "https://bitbucket.org/test-travail/test"},
 				repoRootSubdir: "depth1",
+				scheme:         core.NoScheme,
+				sshusername:    "",
+			},
+		},
+		{
+			urlStr: "git@github.com/shykes/daggerverse.git/ci",
```

Comment:
As mentioned above - I think we should remove this for now. Implicit git urls are messy.

Also note - these are really likely to get confused with SCP-style URLs, which would be `git@github.com:shykes/daggerverse.git/ci`.